### PR TITLE
renamme flow(s) in thread(s) (or warp thread(s))

### PR DESCRIPTION
**Pull request by Stef-SD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,15 +213,15 @@ This way it'll end up with `11` as the final value.
 
 # Definitions
 
-## <a name="parallel-flow"></a>Parallel flow
+## <a name="parallel-thread"></a>Parallel thread
```
Comment (jpl-scale) [inline README.md]: This is not easy. This is the the definition of a _warp_ thread. We need to introduce the fact it is not necessarily a thread in the OS PoV, but could be in another process.

File: README.md [hunk 2]
```diff
@@ -81,16 +81,16 @@ Prints out
 
 When you _warp_ call a function, the [execution context](#execution-context) (i.e. everything
 reachable from the function scope) is captured, then this context and the _warped_ function are
-sent to a distinct execution flow ([parallel flow](#parallel-flow)).
+sent to a distinct execution warp thread ([parallel thread](#parallel-thread)).
 
 Once the [_warp_ task](#warp-task) is finished, the parallel [context](#execution-context) is
-captured and sent back to the main javascript flow. Then, in an event task of the main flow,
+captured and sent back to the main javascript warp thread. Then, in an event task of the main warp thread,
```
[the remark this answers is not in the document]
Comment (Stef-SD) [reply README.md]: It's corrected.

File: README.md (hunk 1 above)
Comment (Stef-SD) [reply README.md]: I introduced a new sentence. Is it ok for you?

File: README.md [hunk 3]
```diff
@@ -81,16 +81,16 @@ Prints out
 
 When you _warp_ call a function, the [execution context](#execution-context) (i.e. everything
 reachable from the function scope) is captured, then this context and the _warped_ function are
-sent to a distinct execution flow ([parallel flow](#parallel-flow)).
+sent to a distinct execution _warp_ thread ([parallel thread](#parallel-thread)).
```
Comment (jpl-scale) [inline README.md]: a distinct and parallel _warp_ thread

File: README.md [hunk 4]
```diff
@@ -213,15 +213,16 @@ This way it'll end up with `11` as the final value.
 
 # Definitions
 
-## <a name="parallel-flow"></a>Parallel flow
+## <a name="parallel-thread"></a>Parallel thread
```
Comment (jpl-scale) [inline README.md]: Why not rename that section _warp_ thread?

File: doc/warp.d.ts [hunk 5]
```diff
@@ -9,23 +9,23 @@ declare module "warp" {
      * Start a parallel execution and provides its result as a Promise to be resolved.   
      *   
      * Perform a call equivalent to `fnct(...args)` in a
-     * [parallel flow](https://github.com/ScaleDynamics/warp/blob/master/README.md#parallel-flow).
+     * [parallel thread](https://github.com/ScaleDynamics/warp/blob/master/README.md#parallel-thread).
```
Comment (jpl-scale) [inline doc/warp.d.ts]: s/parallel/_warp_ thread/

File: doc/warp.d.ts [hunk 6]
```diff
@@ -9,23 +9,23 @@ declare module "warp" {
...
      *   
      * The [Execution context](https://github.com/ScaleDynamics/warp/blob/master/README.md#execution-context)
-     * sent to the parallel flow is captured at the time of the call to `warp.callAsPromise`.
+     * sent to the parallel warp thread is captured at the time of the call to `warp.callAsPromise`.
```
Comment (jpl-scale) [inline doc/warp.d.ts]: underscore

File: README.md (hunk 4 above)
Comment (Stef-SD) [reply README.md]: We define 'threads' in the JavaScript sense then in the warp sens. I preferred to remain more general.

File: doc/warp.d.ts (hunk 6 above)
Comment (Stef-SD) [reply doc/warp.d.ts]: also in comments?

File: README.md [hunk 7]
```diff
@@ -4,7 +4,7 @@
 # Warp
 
 _Warp_ is an API allowing easy parallel execution of javascript code, at the
-same physical time as the main javascript flow and as other flows.
+same physical time as the main javascript thread and as other thread.
```
[the remark this answers is not in the document]
Comment (Stef-SD) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (Stef-SD) [reply README.md]: done

**Pull request merged**